src/main/java/com/github/ogomezso/kafka/connect/soap/source/SoapSourceConnector.java

```diff
@@ -53,15 +53,24 @@ public void start(Map<String, String> map) {
 
   @Override
   public List<Map<String, String>> taskConfigs(int maxTasks) {
-    //TODO Allow a list of request files.
     // Every task will receive one file as config.
     // Every request must be handled by its own task so max tasks must be the number of request records.
-    if (maxTasks != 1) {
-      throw new ConfigException("Task must be exactly one");
-    }
 
-    //TODO Separate Task Config from Source Config. For the moment is exactly the same so don't do now.
-    return Collections.singletonList(settings);
+    ArrayList<Map<String, String>> taskConfigs = new ArrayList<>();
```

Comment:
If I understood well for this iteration every task will take care to exactly one kind of request (1 request file per task) so perhaps we can add a similar validation of the previous one that validate that the number of task is the same of the number of files to process

Reply:
Not sure I get what you mean; however the code changed again in yesterday's midnight commit.  Perhaps there should be a unit test that for a lot of files checks if every file is indeed in one task config?

Reply:
Yes I think we need to create a unit test for creating a connector with more than one files/tasks

Reply:
Well I did create some but they just check if the task configs generated in taskConfigs() do contain a requestMessageFile field, and if that one contains a valid path, and that 
`assertEquals(Math.min(files, maxTasks), actual.size());`